flink-table/flink-sql-client/src/test/resources/sql/function.q

```diff
@@ -15,6 +15,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+ADD JAR '$VAR_PIPELINE_JARS_PATH';
```

Comment:
I think the variable is not suitable here, because it refers to the `pipeline.jars` configuration value. Would be better to use `$VAR_UDF_JAR_PATH` or something similar to make the meaning clear.